apps/hasura.planx.uk/migrations/default/1765484954679_create_view_analytics_planning_data_health/up.sql

```diff
@@ -0,0 +1,18 @@
+CREATE OR REPLACE VIEW analytics_planning_data_health AS (
+SELECT
+    t.id AS team_id,
+    t.name AS team_name,
+    t.slug AS team_slug,
+    ti.has_planning_data AS planning_constraints_enabled_in_planx,
+    ts.reference_code AS planning_data_reference_code,
+    CASE 
+        WHEN ts.reference_code IS NOT NULL
+        THEN FORMAT('https://provide.planning.data.gov.uk/organisations/local-authority:%s', ts.reference_code)
+        ELSE NULL
+    END AS planning_data_link
```

Comment:
`planning_data_link` repeats the same string template pattern we use in the Editor team menus. Currently otherwise hard to "see all team links in one place"